static/app/views/settings/account/accountSecurity/accountSecurityWrapper.tsx

```diff
@@ -17,13 +17,15 @@ type Props = {
 type State = {
   authenticators?: Authenticator[] | null;
   organizations?: OrganizationSummary[];
+  emails: UserEmail[];
 } & AsyncComponent['state'];
 
 class AccountSecurityWrapper extends AsyncComponent<Props, State> {
   getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
     return [
       ['authenticators', ENDPOINT],
       ['organizations', '/organizations/'],
+      ['emails', '/users/me/emails/'],
```

Comment:
You can use `ConfigStore.set('user', user);` to avoid an API call.

Reply:
will do this in a follow up PR / commits if other things need to be addressed. good to know!

Reply:
i actually think this might need to remain as is -- if the user verifies their email then closes the modal we want this to update and call the route each time.